### DISCUSSION TITLE
unable to assign model properties

### DISCUSSION THREAD

**Discussion by Mikeyjpk** [prisma/prisma#18918] [Q&A] [answered]
Im building a airbnb clone while learning nextjs. When i attempt to create a new user, i get a server 500. the user const on line 18 displays the follow. 

https://github.com/Mikeyjpk/airbnb-clone-v1/blob/main/app/api/register/route.ts
![image](https://user-images.githubusercontent.com/116474469/234264793-f6f0d522-4b1e-475b-8225-dabaf8c89308.png)

I believe this is a similar issue to the following post. However the solutions arnt working for me https://github.com/prisma/prisma/discussions/11729

My user model looks like
https://github.com/Mikeyjpk/airbnb-clone-v1/blob/main/prisma/schema.prisma
![image](https://user-images.githubusercontent.com/116474469/234265146-9a9c8161-a0db-4ba0-981c-4dcefd46a96d.png)

the entire project is available
https://github.com/Mikeyjpk/airbnb-clone-v1

Thanks, any advice is appreciated.

**Comment by ludralph** [comment, marked as answer]
Hi @Mikeyjpk 👋 

Thank you for sharing a copy of your project. I have installed it locally and was able to sign up a new user. I did not also see the error you are getting.
This is a screenshot of the code in my editor
<img width="545" alt="image" src="https://user-images.githubusercontent.com/13855608/234270742-27e18b9b-95c9-423b-b21e-cfa1c403fa37.png">

Also this is a copy of the test data i inserted when i used the signup functionality
<img width="645" alt="image" src="https://user-images.githubusercontent.com/13855608/234270978-231c4b9e-6ece-4c36-991f-10796b23520d.png">

Can you restart your IDE and see if you still have the same issue? Also did you run `npx prisma generate` command?

**Reply by Mikeyjpk** [reply]
Thank you so much! i had not run the generate command, once i did and reset my IDE it now seems to be working. Thank you for taking the time to help!

**Reply by ludralph** [reply]
You're welcome @Mikeyjpk. Don't hesitate to open a new discussion if you run into any issues.
